README.md

```diff
@@ -1,66 +1,104 @@
-[gitter-chat]: https://gitter.im/CraneStation/Lobby
-[minimum-rustc]: https://img.shields.io/badge/rustc-1.37+-green.svg
+  <h3>
+    <a href="https://bytecodealliance.github.io/wasmtime/">Guide</a>
+    <span> | </span>
+    <a href="https://bytecodealliance.github.io/wasmtime/contributing.html">Contributing</a>
+    <span> | </span>
+    <a href="https://wasmtime.dev/">Website</a>
+    <span> | </span>
+    <a href="https://bytecodealliance.zulipchat.com/#narrow/stream/217126-wasmtime">Chat</a>
+  </h3>
+</div>
 
-[![build-status]][github-actions]
-[![gitter-chat-badge]][gitter-chat]
-![minimum-rustc]
+## Installation
 
-There are Rust, C, and C++ toolchains that can compile programs with WASI. See
-the [WASI intro][WASI intro] for more information, and the [WASI tutorial][WASI tutorial]
-for a tutorial on compiling and running programs using WASI and wasmtime, as
-well as an overview of the filesystem sandboxing system.
+The Wasmtime CLI can be installed on Linux and macOS with a small install
+script:
 
-Wasmtime passes the [WebAssembly spec testsuite]. To run it, update the
-`tests/spec_testsuite` submodule with `git submodule update --remote`, and it
-will be run as part of `cargo test`.
+```sh
+$ curl https://wasmtime.dev/install.sh -sSf | bash
+```
 
-Wasmtime does not yet implement Spectre mitigations, however this is a subject
-of ongoing research.
+Windows or otherwise interested users can download installers and
+binaries directly from the [GitHub
+Releases](https://github.com/bytecodealliance/wasmtime/releases) page.
 
-[WebAssembly spec testsuite]: https://github.com/WebAssembly/testsuite
-[CloudABI]: https://cloudabi.org/
-[WebAssembly System Interface]: docs/WASI-overview.md
-[WASI intro]: docs/WASI-intro.md
-[WASI tutorial]: docs/WASI-tutorial.md
+## Example
 
-Additional goals for Wasmtime include:
- - Support a variety of host APIs (not just WASI), with fast calling sequences,
-   and develop proposals for additional API modules to be part of WASI.
- - Facilitate development and testing around the [Cranelift] and [Lightbeam] JITs,
-   and other WebAssembly execution strategies.
- - Develop a native ABI used for compiling WebAssembly suitable for use in both
-   JIT and AOT to native object files.
+If you've got the [Rust compiler
+installed](https://www.rust-lang.org/tools/install) then you can take some Rust
+source code:
 
-[Cranelift]: https://github.com/bytecodealliance/cranelift
-[Lightbeam]: https://github.com/bytecodealliance/wasmtime/tree/master/crates/lightbeam
+```rust
+fn main() {
+    println!("Hello, world!");
+}
+```
 
-#### Including Wasmtime in your project
+and compile/run it with:
 
-Wasmtime exposes an API for embedding as a library through the `wasmtime` subcrate,
-which contains both a [high-level and safe Rust API], as well as a C-compatible API
-compatible with the [proposed WebAssembly C API].
+```sh
+$ rustup target add wasm32-wasi
+$ rustc hello.rs --target wasm32-wasi
+$ wasmtime hello.wasm
```

Comment:
not sure if we want to suggest `cargo wasi` and/or have a link for more info

Reply:
I figured this was too prominent of a position to mention `cargo wasi`, and tbh I already felt somewhat uncomfortable bringing `rustc` into the picture here. There's not really a canonical repository/registry of wasm files though to draw from so I figured it'd be best to keep it as simple as possible here to avoid complicating the "hello world" instructions